templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Car</h1>
+  <p>Are you sure to delete the car: {{ car }}</p>
+  <form action="" method="post" novalidate>
+  {% csrf_token %}
+  <input type="submit" value="Yes, delete" class="btn btn-danger">
+  <input type="button" onclick="history.back();" value="Cancel" class="btn btn-primary">
```

Comment:
The Cancel button is correctly implemented to lead back to the previous page using 'history.back();'. This fulfills the requirement to have a Cancel button apart from the Delete one, as specified in the checklist.